packages/fluentui/react-northstar/src/utils/shouldPreventDefaultOnKeyDown.ts

```diff
@@ -10,7 +10,10 @@ export function shouldPreventDefaultOnKeyDown(e: KeyboardEvent | React.KeyboardE
 
   const matchesByKey = code === SpacebarKey || code === EnterKey;
   const ignoredByTag =
-    target?.tagName === 'INPUT' || target?.tagName === 'TEXTAREA' || target?.isContentEditable === true;
+    target?.tagName === 'INPUT' ||
+    target?.tagName === 'TEXTAREA' ||
+    target?.isContentEditable === true ||
+    (target?.tagName === 'A' && code !== SpacebarKey);
```

Comment:
Let's restore this one and handle it earlier

```tsx
if (target?.tagName === 'A') {
  return code === SpacebarKey
}
```

Reply:
Done